src/server/modules/mailer/index.js

```diff
@@ -0,0 +1,6 @@
+import transporter from './transporter';
+import Feature from '../connector';
+
+export default new Feature({
+  createContextFunc: () => ({ transporter })
```

Comment:
Why to name in `transporter` here, instead of `mailer` or something like that?

Reply:
done